src/Revamp.re

```diff
@@ -2,7 +2,7 @@ open Rebase;
 
 [@bs.set] external _setLastIndex : (Js.Re.t, int) => unit = "lastIndex";
 [@bs.send.pipe : string] external _replace : (Js.Re.t, string => string) => string = "replace";
-
+[@bs.send.pipe: string] external _splitByRe: Js.Re.t => array(string) = "split";
```

Comment:
This is unsound, so all this'll do is reintroduce the unsoundness that was fixed in BuckleScript. I'll show what I meant below.